propagators/aws/xray/README.MD

```diff
@@ -0,0 +1,10 @@
+# AWS X-Ray Propagator/IDGenerator
+
+This package contains an AWS X-Ray compatible propagator along
+with an x-ray compatible IDGenerator.
+
+## `traceIdRatioSampler` and `x-ray IDGenerator` compatability
+
+It is a general suggestion to `NOT` use the `traceIDRatioSampler` while also
+using the `x-ray IDGenerator`. The non-random nature of building an x-ray traceId
+may lead to unexpected sampling results.
```

Comment:
How about adding a `Usage` section?

Reply:
I think a `usage` section would be useful but also don't want to hold back this PR as I may not have time to get to it immediately. I believe adding the warning for the incompatibilities between sampler/generator warrant this going through. If desired, I can open an issue to expand the README with a usage section.